kernel/objects/common/objects/ISchedulable.hh

```diff
@@ -65,6 +65,8 @@ namespace mythos {
     
     virtual void saveState() = 0;
 
+    virtual bool hasPriority() { return false; }
```

Comment:
Maybe at least the interface should be able to hande different priorities?
We could define some integer constants like `HIGH_PRIORITY, DEFAULT_PRIORITY` and have defined behaviour only for these. I imagine we want to have `LOW_PRIORITY` somewhen in the future.

Reply:
added comment. 
integer constants sound good, but I don't think it is necessary.